release/ThirdPartyNotices.txt

```diff
@@ -0,0 +1,312 @@
+THIRD-PARTY SOFTWARE NOTICES AND INFORMATION
+
+React Native Tools  is based on or incorporates material from the projects listed below (Third Party OSS). The original copyright notice and the license under which Microsoft received such Third Party OSS, are set forth below. Such licenses and notices are provided for informational purposes only. Microsoft licenses the Third Party OSS to you under the licensing terms for the Microsoft product or service. Microsoft reserves all other rights not expressly granted under this agreement, whether by implication, estoppel or otherwise. 
+
+
+1.	amdefine (https://github.com/jrburke/amdefine)
+2.	applicationinsights (https://github.com/Microsoft/ApplicationInsights-Home)
+3.	extract-opts  (https://github.com/bevry/extract-opts)
+4.	flatten-source-map (https://github.com/DavidSouther/flatten-source-map)
+5.	options (https://github.com/einaros/options.js)
+6.	q (https://github.com/kriskowal/q)
+7.	semver (https://github.com/npm/node-semver)
+8.	source-map (https://github.com/mozilla/source-map)
+9.	strip-json-comments (https://github.com/sindresorhus/strip-json-comments)

```

Comment:
Note: We don't actually depend on this right now, but in the exponent integration we do, so I'm including this here rather than messing with licenses etc.